src/backend/joanie/core/models/certifications.py

```diff
@@ -117,7 +136,7 @@ def get_document_context(self, language_code=None):
 
         language_settings = get_language_settings(language_code or get_language())
         organization = self.order.course.organization
-        owner = self.order.owner
+        owner = self.owner
 
         base_context = {
```

Comment:
At first I wondered: should we add a database constraint to ensure that self.owner == self.order.owner and self.certificate_definition == self.order.certificate_definition for cases when self.order is not null? 

Then I saw that the "order" field is required so I'm not sure anymore why we need to repeat these 2 fields on the cerificate model? It does not reduce entropy :man_shrugging:

Reply:
Indeed ! I wonder if I could create a property `owner` but I think it could be more confusing than relevant.